packages/web-react/docs/stories/feature-flags/Overview.stories.mdx

```diff
@@ -0,0 +1,57 @@
+import { Meta } from '@storybook/blocks';
+
+<Meta title="Experimental/Feature Flags" />
+
+# Feature Flags
+
+## Table of Contents
+
+- [Current feature flags](#current-feature-flags)
+- [Turning on feature flags in SASS](#turning-on-feature-flags-in-sass)
+
+The `@lmc-eu/spirit-web-react` codebase ships with a number of feature flags. These feature
+flags enable new behavior and styling, allowing you to opt-in to new (and
+sometimes breaking) changes while using the current version. A feature flag may
+be configured in JavaScript, SASS, or both.
+
+This section in the Storybook documents each feature flag that is available and
```

Comment:
will we actually keep another list of Feature Flags here? Should we just embed some other MD, where it will be documented centrally?

Reply:
Good point. I think we can use this file as a draft for some MD or something else. Do you have any idea where to store it?

Reply:
Why not in the root docs folder? And linked in the main readme and/or in packages readmes?

Reply:
I would keep it there and we can embed it here, like we embed the component readmes.

Reply:
Ok.

Reply:
Great idea. We should do this with other files, I think :-)